lib/jekyll/assets/liquid/tag/proxied_asset.rb

```diff
@@ -109,8 +139,12 @@ def proxy_file
 
           private
           def cache_file
-            @_cached = File.file?(filename)
-            return @_cached if @_cached
+            glob = Dir.glob("#{filename.chomp(File.extname(filename))}.*")
+            @_cached = !glob.empty?
+            if @_cached
+              self.content_type = _mime_for(File.extname(glob.first))
```

Comment:
When you move to Pathutil you'll move this to `glob.first.extname` only.  Skipping `File`